cmd/post-github/main.go

```diff
@@ -59,7 +54,7 @@ func runGH(
 
 Stress build found a failed test:
 
-%s`, sha, makeCodeLike(test.Message))
```

Comment:
Agreed, and yet there appears to be no sane way to do it. It's also the case that "```" is only recognized as closing a code block if it's on a line by itself...and then it's not clear how to escape it without changing it. Gonna punt.